react/components/BuyButton/index.js

```diff
@@ -182,33 +189,60 @@ export const BuyButton = ({
     }, 500)
   }
 
-  return (
-    <Fragment>
-      {!skuItems ? (
-        <ContentLoader />
-      ) : (
-        <Button
-          block={large}
-          disabled={
-            disabled ||
-            !available ||
-            (orderFormContext && orderFormContext.loading)
-          }
-          isLoading={isAddingToCart}
-          onClick={e => shouldAddToCart && handleAddToCart(e)}
-        >
-          {available ? (
-            children
-          ) : (
-            <FormattedMessage id="store/buyButton-label-unavailable">
-              {message => (
-                <span className={handles.buyButtonText}>{message}</span>
-              )}
-            </FormattedMessage>
+  const handleClick = e => {
+    dispatch({ type: 'SET_BUY_BUTTON_CLICKED', args: { clicked: true } })
+    if (!skuSelector.areAllVariationsSelected) {
+      // TODO show toast or something
```

Comment:
Is this a registered issue?

Reply:
> Is this a registered issue?

No... I was thinking about this right now and we probably won't do anything in this case, so I'll just remove this code, thx